content/source/docs/cloud/migrate/index.html.md

```diff
@@ -1,124 +1,32 @@
-
-~> **Important:** These instructions are for migrating state in a basic working directory that only uses the `default` workspace. If you use multiple [workspaces][cli-workspaces] in one working directory, the instructions are different; see [Migrating State from Multiple Terraform Workspaces](./workspaces.html) instead.
-
--> **API:** See the [State Versions API](../api/state-versions.html). Be sure to stop Terraform runs before migrating state to Terraform Cloud, and only import state into Terraform Cloud workspaces that have never performed a run.
-
-## Step 1: Ensure Terraform ≥ 0.11.13 is Installed
-
-To follow these instructions, you need Terraform 0.11.13 or later on the workstation where you are performing the migration. In older versions, [the `remote` backend][remote-backend] is either absent or only partially supported.
-
-## Step 2: Gather Credentials, Data, and Code
-
-Make sure you have all of the following:
-
-- The location of the VCS repository for the Terraform configuration in question.
-- A local working copy of the Terraform configuration in question.
-- The existing state data. The location of the state depends on which [backend][] you've been using:
-    - If you were using the default `local` backend, your state is a file on disk. You need the original working directory where you've been running Terraform, or a copy of the `terraform.tfstate` file to copy into a fresh working directory.
-    - For other backends, you need the path to the particular storage being used (usually already included in the configuration) and access credentials (which you usually must set as an environment variable).
-- A Terraform Cloud user account.
-
-    This account must have permission to manage workspaces for the organization. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
-
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
-
-You also need to authenticate Terraform with Terraform Cloud.  If you're using Terraform 0.12.21 or later, you can use the `terraform login` command. Alternatively, you can create a [user API token][user-token] and [manually configure credentials in the CLI config file][cli-credentials].
-
-## Step 3: Stop Terraform Runs
-
-However you currently run Terraform to manage this infrastructure, stop. Let any current runs finish, then make sure there won't be more.
-
-This might involve locking or deleting CI jobs, restricting access to the state backend, or just communicating very clearly with your colleagues.
+## Requirements
 
-## Step 4: Prepare Your Terraform Working Directory
+Stop all Terraform runs before migrating state files. You should also only migrate state files into Terraform Cloud workspaces that have never performed a run.
```

Comment:
I think this applies to the current workspace the state files apply to (as opposed to runs in your TFC org)?

Reply:
yesss I think you're right. I'm going to specify - thank you!

Reply:
Actually i'm confused - @korinne or @chrisarcand This was in the docs before I came in but did we mean "stop all runs locally"? The reason why is it doesn't make sense to tell users to do this in TFC if they have to migrate state into new workspaces that have never performed a run anyway... Unless the guidance actually IS to make it so that NO runs are happening in your TFC org (which seems disruptive).

Reply:
I think this just means to stop all runs (local and/or remote) before you're migrating state files. This means that you should either wait for any running `apply` to finish, or to cancel them

Reply:
okay I pushed up a change that I think helps clarify @im2nguyen will you take a look when you do your review? :-)

Reply:
LGTM! Would like @korinne or @chrisarcand to confirm before you resolve this tho 😄

Reply:
Chris has confirmed that this looks good and we made a tiny tweak to the language as well. I'll push up the change and then I think we should be good to go!! Thank you!